vignettes/articles/modify-functions.Rmd

```diff
@@ -0,0 +1,197 @@
+---
+title: "Modifier Functions"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(gtsummary)
+theme_gtsummary_compact()
+```
+
+## Introduction
+
+Oftentimes when a table is created using {gtsummary} the default formatting does not exactly match a user's preferred formatting. To make modifications to tables after the table has been built, the `modify_*()` functions were created.
+
+In the following sections we briefly outline each of these functions that can be used to modify gtsummary tables.
+
+Many of the modifier functions share the common `columns` argument. When this argument is required, the column names provided should match their names in the table's `table_body` element (i.e. `x$table_body`). These column names can be printed to the console by running `show_header_names(x)`.
+
+When the `rows` argument is available for a modifier function, a subset of rows in the table on which the function should operate can also be specified via a predicate expression selecting rows from `x$table_body`. See the function documentation for details.
+
+We will begin by crafting a basic table to summarize a regression model. In the following sections, we will showcase how to use a variety of table modifier functions by applying them to this basic table.
+
+```{r tbl}
+x <- glm(response ~ age + grade, trial, family = binomial()) |>
```

Comment:
Perhaps we can change the motivating example to something our colleagues will see more often? Perhaps `tbl_summary() |> add_difference()`? We can highlight that the primary stat columns are already character, and functions like `modify_fmt_fun()` are typically not useful for them, but this table also has columns `estimate`, `conf.low`, `conf.high`, `p.value` etc that have modifying functions.

Reply:
But this won't have a way to showcase the `modify_missing_symbol()` function. maybe a one-off example with that near the end is enough?

Reply:
Example updated